content/reference/endpoints/rentals.md

```diff
@@ -194,7 +194,7 @@ Name              | Type    | Default | Description
 amenities         | String  |         | List of comma separated amenities to include.
 bb_ne             | Float   |         | Rentals located between `bb_sw` & `bb_ne`. Format `latitude, longitude`.
 bb_sw             | Float   |         | Rentals located between `bb_sw` & `bb_ne`. Format `latitude, longitude`.
-bookable_online   | Boolean | blank   | Rentals instantly bookable if `true` or only non instantly bookable if `false`. Leave blank to get all rentals.
+bookable_online   | Boolean | blank   | `true` to get rentals that require no human interaction for booking, `false` to get those that do.
```

Comment:
I would had the term `instant bookable` as well, it's very common in the industry.